src/cpp/server/csds/csds.cc

```diff
@@ -87,7 +86,7 @@ Status ClientStatusDiscoveryService::FetchClientStatus(
     return Status(static_cast<StatusCode>(s.status().raw_code()),
                   s.status().ToString());
   }
-  *response->add_config() = std::move(s.value());
+  *response = s.value();
```

Comment:
This should use `std::move()` to avoid an unnecessary copy.

Reply:
Done